LocalReferenceLinkRel/explainer.md

```diff
@@ -148,6 +145,27 @@ the shadow root where they are defined, as illustrated by the following examples
   Styles defined inside the sibling Shadow Root are not applied, so "Inside Sibling Shadow DOM" is not blue.
 </p>
 
+These scoping behaviors do not apply to [Declarative CSS Modules](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/ShadowDOM/explainer.md)
+because modules always have global scope. Shadow DOM scoping could be modified though other means, as discussed
+in [this thread](https://github.com/whatwg/html/issues/11364). All of the suggestions in that thread would
+allow for this feature to work with any shadow root, regardless of scope.
+
+### Key Differences Between This Propsoal And Declarative CSS Modules
+
+Both this proposal and [Declarative CSS Modules](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/ShadowDOM/explainer.md)
+allow authors to share inline CSS with Shadow Roots. There are some key differences in both syntax and
+behaviors, as illustrated in the following table:
+
+| | Local Reference Link Rel | Declarative CSS Modules | 
+| :---: | :---: | :---: |
+| Scope | Standard DOM scoping | Global scope |
+| Identifier syntax | Standard HTML IDREF | Module identifier |
+| Integration with @sheet | ✅ Yes | ❌ No |
+| Uses existing HTML concepts | ✅ Yes | ❌ No |
```

Comment:
Is there a way CSS Modules could end up being closer to HTML concepts if we adjust the syntax?

Reply:
We could, but then it would diverge from Imperative CSS Modules. Let's sync with @dandclark once he's back.

Reply:
Per the discussion earlier, I converted all examples to `<style>` and made the `identifier` examples look like regular IDREF and less URL-like.